docs/docs/troubleshooting.md

```diff
@@ -58,28 +58,30 @@ profile. To select the `performance` profile:
 echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
 ```
 
-## Switchroot
+## Switchroot/Jetson setup
 
-If you are running switchroot Linux on your Nintendo Switch, you can install
-xemu from the PPA as described in in the [Download](download.md) page. However,
-when running xemu you may see the following error:
+If you are running Switchroot/L4T Ubuntu on your Nintendo Switch or Jetson device, you can install
+xemu from the PPA as described in in the [Download](download.md) page. However, this is generally **not recommended** for performance reasons.
```

Comment:
* I don't really want to push the official ppa as being "not recommended" and instead suggesting people use something that's marked as *use at own discretion*. xemu generally is to be used at your own discretion anyway.
* Where are the data behind the "slight speed boost" claim?
* Performance is pretty bad on switch generally... which games in xemu are actually playable when building with platform tuning?
* We can suggest your alternative build script that builds with tuning flags for the platform, but let's at least make it a separate subsection.
  * Also you can build a deb for the platform instead of duplicating the debian install behavior, which is preferable for updates, etc.

Reply:
> * I don't really want to push the official ppa as being "not recommended" and instead suggesting people use something that's marked as _use at own discretion_. xemu generally is to be used at your own discretion anyway.
> * We can suggest your alternative build script that builds with tuning flags for the platform, but let's at least make it a separate subsection.

Sounds good, I'll rewrite that when I get the chance.

> * Where are the data behind the "slight speed boost" claim?
> * Performance is pretty bad on switch generally... which games in xemu are actually playable when building with platform tuning?

I don't have a message link, but I remember `Marco Loves 360#3902 ` confirming last year in the Discord server that there was a small bump in FPS when manually compiling. Unfortunately I haven't been able to test this myself, but I'm hoping the gap in performance between the PPA and source building increases long term as the emulator gets more ARM optimizations... the CPU is a massive bottleneck for Tegra hardware like the Switch.

>   * Also you can build a deb for the platform instead of duplicating the debian install behavior, which is preferable for updates, etc.

That's an option, but this approach is more hands off - updates are rebuilt with the Megascript itself by selecting the [Auto Updater](https://github.com/cobalt2727/L4T-Megascript/blob/master/scripts/update.sh) option in the GUI.

Relevant bits:

```bash
XemuUserInput="no" #set this variable to a default response
if test -f /usr/local/bin/xemu; then #check if Xemu's been compiled on the device
        description="Do you want to update Xemu? (May take 5 minutes to 2 hours)"
        table=("yes" "no")
        userinput_func "$description" "${table[@]}"
	XemuUserInput="$output"
fi #create a GUI (or a RetroPie-style terminal menu if connecting over SSH) to ask the user whether they want to update Xemu

# ... running other stuff

if [[ $XemuUserInput == "yes" ]]; then
	echo "Updating Xemu..."
	sleep 5
	bash -c "$(curl -s https://raw.githubusercontent.com/$repository_username/L4T-Megascript/$repository_branch/scripts/games_and_emulators/xemu.sh)" || exit $?
else
	echo "Skipping Xemu update..."
fi
```

Reply:
I can attest that manually compiling xemu on switch hardware does net better performance, which in a situation where the switch is already power limited, every little bit of perf you can get helps

Reply:
How big of a difference is there? In the test I mentioned above I remember he only said it was about 1-4 FPS faster on Halo 1.